variables.tf

```diff
@@ -84,3 +84,15 @@ variable "descriptions" {
   description = "List of descriptions for the created service accounts (elements default to the value of `description`)"
   default     = []
 }
+
+variable "rotation_days" {
+  type        = number
+  description = "Time key rotation"
+  default     = 30
+}
+
+variable "rotation_key" {
+  type        = bool
+  description = "Active key rotation"
+  default     = false
+}
```

Comment:
Can we use just one variable to control this? If rotation_days is null (by default) its not active

Reply:
yes, is valid